cfg/eclipse-keys.lua

```diff
@@ -0,0 +1,100 @@
+-- In your user / system configuration file simply type: include "eclipse-keys.lua"
```

Comment:
Yes, `include` should be used as it does things that `require` doesn't do in this context: searches locally and relatively to the config where it's called plus in one places known to store config, provides the expected environment, and few other things.